Misc/NEWS.d/next/Library/2018-02-26-13-16-36.bpo-32713.55yegW.rst

```diff
@@ -0,0 +1 @@
+Fixed tarfile.itn handling of out-of-bounds float values
```

Comment:
Please add a period at the end of the sentence and "Patch by *yourname*."

Reply:
done :+1: